### PR TITLE
[docs] Polish a bit the docs

### PR DESCRIPTION
**Pull request by oliviertassinari**
Initially, wanted to fix:

Screenshot 2023-04-22 at 20 44 28

https://app.ahrefs.com/site-audit/3833384/43/data-explorer?columns=pageRating%2Curl%2Ctraffic%2ChttpCode%2Cdepth%2CmetaDescription%2CmetaDescriptionLength%2CnrMetaDescription%2Ccompliant&filterId=2864d4a78dbccdda9cf9629aa273bd61&issueId=c64d56c9-d0f4-11e7-8ed1-001e67ed4656

But went a bit above, doing more cleanup.

### CODE REVIEW

File: docs/data/pages.ts [hunk 1]
```diff
@@ -20,11 +20,11 @@ const pages: MuiPage[] = [
       },
       {
         pathname: '/toolpad/connecting-to-datasources/serverside-http-request',
-        title: 'serverside HTTP request',
+        title: 'Serverside HTTP requests',
```
Comment (oliviertassinari) [inline docs/data/pages.ts:23]: To match the h1 of the page.

File: docs/data/toolpad/building-ui/canvas-and-inspector.md [hunk 2]
```diff
@@ -22,13 +22,13 @@ The canvas will highlight the location where a component can be dropped. Release
 
 Click any component in the canvas to select it. An overlay indicates the selected components.
 
-<img src="/static/toolpad/docs/building-ui/ui-2.png?v=0" width="230px" alt="Canvas controls" />
+<img src="/static/toolpad/docs/building-ui/ui-2.png?v=0" width="230" alt="Canvas controls" />
```
Comment (oliviertassinari) [inline docs/data/toolpad/building-ui/canvas-and-inspector.md:25]: e.g. https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/

File: CHANGELOG.md [hunk 3]
```diff
@@ -359,7 +359,7 @@ New date picker component, Make switching views part of undo/redo cycle.
 - &#8203;<!-- 07 -->Datepicker format handling changes (#1504) @bytasv
 - &#8203;<!-- 04 -->Undo/redo through different views (#1417) @apedroferreira
 - &#8203;<!-- 03 -->Make sure npm package is splitted out correctly (#1502) @Janpot
-- &#8203;<!-- 02 -->Date picker component (#1499) @bytasv
+- &#8203;<!-- 02 -->Date Picker component (#1499) @bytasv
```
Comment (oliviertassinari) [inline CHANGELOG.md:362]: The capitalization convention for the component names.

File: docs/data/toolpad/connecting-to-datasources/queries.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
 # Queries
 
-<p class="description">Toolpad allows you to bring remote data as state to the page and allows your UI to bind itself to this state.</p>
+<p class="description">A query allows to bring remote data as "state" to the page and allows your UI to bind itself to that state.</p>
 
-Be it a database table or an external API, most data we want to display in Toolpad pages originates serverside. Toolpad offers mechanisms to bring this serverside data to the page. The most common way is through **queries**. To configure a query we need to follow these steps:
```
Comment (oliviertassinari) [inline docs/data/toolpad/connecting-to-datasources/queries.md:5]: The use of "we" is rarely appropriate.

Comment (Janpot) [reply docs/data/toolpad/connecting-to-datasources/queries.md:5]: alternatively

"most data displayed on toolpad pages"

"to configure a query, follow these steps"

**Review by Janpot — APPROVED** [review]

File: docs/data/toolpad/connecting-to-datasources/queries.md [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
+Be it a database table or an external API, most data you want to display in Toolpad pages originates serverside. Toolpad offers mechanisms to bring this serverside data to the page. The most common way is through **queries**. To configure a query you need to follow these steps:
```
Comment (oliviertassinari) [inline docs/data/toolpad/connecting-to-datasources/queries.md:5]:
```suggestion
Be it a database table or an external API, most data you want to display in Toolpad pages originates serverside.
Toolpad offers mechanisms to bring this serverside data to the page.
The most common way is through **queries**, to configure a query you need to follow these steps:
```

**Pull request merged**